accepted/2024/[TS]PMUI-Readme-rendering.md

```diff
@@ -0,0 +1,77 @@
+# Markdown Readme Rendering in PM UI
+
+- Jonatan Gonzalez ([jgonz120](https://github.com/jgonz120)) 
+- Issue [#12583](https://github.com/NuGet/Home/issues/12583) <!-- GitHub Issue link -->
+- [Feature Spec](https://github.com/NuGet/Home/blob/7943122dffa435f4daeee600efcc5b744cd2e97e/accepted/2023/PMUI-Readme-rendering.md)
+
+## Summary
+
+We want to add update the PM UI to render the ReadMe.md file while browing packages. 
+
+## Motivation 
+
+We want help our customers understand what a package does and if it will be helpful to them. This could also increase ReadMe adoption if developers know their users will see the ReadMe. 
+
+## Explanation
+
+### Functional explanation
+When a package is selected we will determine if a ReadMe file exists and if it does we'll render it in the PM UI. 
+
+The PM UI will be updated to have tabs for the Package Details and the the ReadMe. This UX will be displayed for both the Browse and installed tabs.
+![Alt text](https://github.com/NuGet/Home/assets/89422562/81b24877-f12f-4783-905c-4a155d3c7693)
+<!-- Explain the proposal as if it were already implemented and you're teaching it to another person. -->
+<!-- Introduce new concepts, functional designs with real life examples, and low-fidelity mockups or  pseudocode to show how this proposal would look. -->
+
+### Technical explanation
```

Comment:
I added some questions regarding this. I found that in my NuGet folder there were ReadMe.md files in the root of the package and in the _content folder for the package. 

As for getting the ReadMe from the API. Is the API we use well documented? I imagine it would be since it would need to be implemented by anyone who wants to implement a source right?